source/2019-12-20-octane-is-here.md

```diff
@@ -0,0 +1,340 @@
+
+const Person = EmberObject.extend({
+  firstName: null,
+  lastName: null,
+  age: null,
+  country: null,
+
+  fullName: computed('firstName', 'lastName', function() {
+    return `${this.firstName} ${this.lastName}`;
+  }),
+
+  description: computed('fullName', 'age', 'country', function() {
+    return `${this.fullName}; Age: ${this.age}; Country: ${this.country}`;
+  })
+});
+
+let captainAmerica = Person.create({
+  firstName: 'Steve',
+  lastName: 'Rogers',
+  age: 80,
+  country: 'USA'
+});
+
+captainAmerica.description; // "Steve Rogers; Age: 80; Country: USA"
+captainAmerica.set('firstName', 'Steven');
+captainAmerica.description; // "Steven Rogers; Age: 80; Country: USA"
+```
+
+This design makes it harder to break up a computed property into smaller functions, because the computed property still needs to enumerate all properties that it used, no matter where they're used. In practice, this means that, in Classic Ember, you break up computed properties into more computed properties, which works well but is somewhat restrictive.
+
+#### After: Tracked Properties
+
+Octane's reactivity model, tracked properties, have a much lighter footprint.
+
+```js
+class Person {
+  @tracked firstName;
+  @tracked lastName;
+  @tracked age;
+  @tracked country;
+
+  constructor({ firstName, lastName, age, country }) {
+    this.firstName = firstName;
+    this.lastName = lastName;
+    this.age = age;
+    this.country = country;
+  }
+
+  get fullName() {
+    return `${this.firstName} ${this.lastName}`;
+  }),
+
+  get description() {
+    return `${this.fullName}; Age: ${this.age}; Country: ${this.country}`;
+  })
+}
+
+let captainAmerica = new Person({
+  firstName: 'Steve',
+  lastName: 'Rogers',
+  age: 80,
+  country: 'USA'
+});
+
+captainAmerica.description; // "Steve Rogers; Age: 80; Country: USA"
+captainAmerica.firstName = "Steven";
+captainAmerica.description; // "Steven Rogers; Age: 80; Country: USA"
+```
+
+You start with a normal JavaScript class and annotate any fields that could affect the DOM with `@tracked`. You don't need to annotate getters or functions, so you can break up your code however you want.
+
+A neat thing about the tracked properties reactivity model is that if you remove the `@tracked` annotation, the code works exactly the same. The only thing that changes if you add `@tracked` is that if you make changes to the property, any part of the DOM that used that property as part of its computation will correctly update.
+
+## A Focus on Documentation
+
+Octane is more than just new features. It's also a focus on refreshing the documentation to show users how to build applications in the Octane way.
+
+### Fully Refreshed Tutorial and Component Guides
+
+The tutorial is the first way that people learn how to build Ember applications. Ember 3.15 fully refreshed the [Super Rentals Tutorial](https://guides.emberjs.com/v3.15.0/tutorial/part-1/) so that it's written in Octane style.
```

Comment:
```suggestion
The tutorial is the first way that people learn how to build Ember applications. Ember 3.15 fully refreshed the [Super Rentals Tutorial](https://guides.emberjs.com/v3.15.0/tutorial/) so that it's written in Octane style.
```